### DISCUSSION TITLE
Linux mint 20.2 not recognizing my Hottop kn-8828b-2k+

### DISCUSSION THREAD

**Discussion by carterscoffee** [artisan-roaster-scope/artisan#728] [Q&A]
I am trying to get my Hottop roster to connect to my laptop so I can use artisan. No matter what I do my laptop does not recognize my roaster being plugged in via USB. I am using Linux mint 20.2

**Comment by RickGroszkiewicz** [comment]
Sounds like you have not chosen the correct USB port. On my linux laptops with Mint / Cinnamon, the port is labeled as FT230 (?). 
I'm not 100% sure of the name, since I forgot to make a screen shot for my own setup notes:
"http://www.softwarepolish.com/personal/Artisan-Reinstalled-Linux-Mint-20/album/index.html"

**Reply by RickGroszkiewicz** [reply]
On my laptops, the port is labeled as FT230X Basic UART. Here is an artisan 2.4.2 screen shot:

![Hottop B-2K+ does show correct port when attached](https://user-images.githubusercontent.com/25457554/143656715-3539eb6d-0587-457a-9c02-0f48c8698694.png)

**Reply by Ralf132** [reply]
Hi, I am pretty new but having the same issue. I have recently installed Artisan (V 3.0.2) on my tablet PC/laptop running with Linux (Linux Mint 21.3 Cinnamon) and I have selected the USB port as shown in your screenshot. But with any further click or activity the status line above says immediately "Hottop disconnected". I guess it could be USB access problems due to Linux but I am not an expert with Linux to allow Hottop to access via the mentioned USB. Maybe someone has a solution for a non expert Linux user?! Thx

**Reply by MAKOMO** [reply]
Did you read Step 3 of the [Installation Instructions]8https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/Installation.md)? Your user account might not have the right to access the USB hardware.

**Reply by Ralf132** [reply]
Yes, I was reading the instructions when I installed Artisan but was not sure which part of step 3 is relevant to me. Need to add that I am a very basic and new Linux user. BUT (!!!!!!!!!!!) with your hint I checked again step 3, added myself to the "dialout" group AND IT IS WORKING NOW !!! I am so happy. Thank you so much Marko - 5 star information which resolved my issue. THX!
